test/test_model.py

```diff
@@ -767,6 +768,40 @@ def test_storage_tool_errors(self):
                 run()
             self.assertEqual(fake_script_calls(self, clear=True), calls)
 
+    def test_network_get(self):
+        network_get_out = ('{"bind-addresses":[{"mac-address":"","interface-name":"",'
+                           '"addresses":[{"hostname":"","value":"192.0.2.2","cidr":""}]}]'
+                           ',"egress-subnets":["192.0.2.2/32"],"ingress-addresses":["192.0.2.2"]}')
```

Comment:
Not the usual quoting here and elsewhere in the PR, plus point by @chipaca is still open